### DISCUSSION TITLE
Product Variants

### DISCUSSION THREAD

**Discussion by magedfathi** [Blair2004/NexoPOS#415] [Ideas]
I would like to suggest having product variants like i am selling Tea when i am pressing on tea product it will show the different size Medium Large Small each one with its own price

Tea
Small 2 QR
Medium 3QR
Large 4 QR

**Comment by Blair2004** [comment]
Hi, 
Thank you for your consideration. This can also be performed using the modifiers.
Concretely, you can create a size group and assign it to a product, then create a "topping" group for additional ingredients. 

This belongs to Gastro 4.x and cannot be merged to NexoPOS 4.x

**Reply by magedfathi** [reply]
Noted
